src/main/java/uk/gov/companieshouse/officerfiling/api/controller/OfficerFilingDataControllerImpl.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.companieshouse.officerfiling.api.controller;
+
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.officerfiling.api.exception.ResourceNotFoundException;
+import uk.gov.companieshouse.officerfiling.api.model.entity.OfficerFiling;
+import uk.gov.companieshouse.officerfiling.api.service.OfficerFilingService;
+
+@RestController
+@RequestMapping("/private/transactions/{transId}/officers")
+public class OfficerFilingDataControllerImpl implements OfficerFilingDataController {
+    public static final String VALIDATION_STATUS = "validation_status";
+    private final OfficerFilingService officerFilingService;
+    private final Logger logger;
```

Comment:
sonar has picked up that this logger is unused.
I've added logging locally and can push here if you like..